File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.cs
using System;

public class Solution 
{
    public int solution(int[] numbers)
    {
        int result = 0;
        for(int i=0; i<=9; i++)
        {    
            if(!numbers.Contains(i))
            {
                result += i;
            }
        }
        return result;
    }
}

File: 프로그래머스/1/86051. 없는 숫자 더하기/없는 숫자 더하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> numbers)
{
    int ans = 0;
    for(int i=0; i<=9; i++)
    {
        bool isBool = false;
        for(int &num : numbers)
        {
            if(i == num)
            {
                isBool = true;
            }
        }
        if(!isBool)
        {
            ans += i;
        }
    }
    return ans;
}